DEVELOPER-QUICKSTART.md

```diff
@@ -0,0 +1,126 @@
+
+# Figures Developer Quickstart Guide
+
+## 1. Overview
+
+This document guides you through getting Figures setup on your development machine.
+
+Requirements are that you have Python 2.7.x installed on your development machine. Hawthorn devstack uses 2.7.12. It is also highly recommended that you install [virtualenv](https://virtualenv.pypa.io/).
+
+## 2. Create a virtualenv
+
+You should use a Python virtualenv to run Figures standalone development mode. This avoids adding packages to the global or user space python instances.
+
+### Resources
+
+Here are links to help guide setting up virtualenv
+
+* https://virtualenv.pypa.io/en/stable/installation/
+* https://virtualenv.pypa.io/en/stable/userguide/
+* https://docs.python-guide.org/dev/virtualenvs/#lower-level-virtualenv
+
+## 3. Clone Figures, install dependencies, and test
+
+On your development machine, open a terminal (command line shell) and create or navigate to the directory for which you want to install Figures. This will be the parent directory for your Figures project. Example:
+
+**These instructions assume you are running a Python virtualenv**
+
+
+In the terminal, run the following:
+
+```
+git clone -b hawthorn-upgrade https://github.com/appsembler/figures.git
+cd figures
+pip install -r devsite/requirements/hawthorn.txt
+```
+
+now we'll run [pytest](https://docs.pytest.org/) to make sure that the tests pass:
+
+```
+pytest
+```
+
+## 4. Set up Figures standalone server
+
+Navigate to your Figures workspace (you will be here if you just completed the previous step)
+
+Run the following:
+
+```
+cd devsite
+./manage.py migrate
+./manage.py createsuperuser
+./manage.py seed_data
```

Comment:
Minor issue: leaving the `Username` empty throws an error instead of using the default value.

```
$ ./manage.py createsuperuser
./manage.py seed_dataUsername (leave blank to use 'laoqui'):
Error: Enter a valid username. This value may contain only English letters, numbers, and @/./+/-/_ characters.
```

Reply:
@lgfa29 Did you run the commands in one copy and paste for both commands or ran them one at a time?

Reply:
I created this ticket: Either an issue in the command or the documentation https://github.com/appsembler/figures/issues/85